en/02_Developer_Guides/12_Search/01_Searchcontext.md

```diff
@@ -77,15 +76,11 @@ class MyDataObject extends DataObject
 }
 ```
 
-[notice]
-See the [SearchFilter](../model/searchfilters) documentation for more information about filters to use such as the
-`GreaterThanFilter`.
-[/notice]
+> [!WARNING]
+> In case you need multiple contexts, consider name-spacing your request parameters by using `FieldList->namespace()` on
+> the `$fields` constructor parameter.
 
-[notice]
-In case you need multiple contexts, consider name-spacing your request parameters by using `FieldList->namespace()` on
-the `$fields` constructor parameter.
-[/notice]
+See the [SearchFilter](../model/searchfilters) documentation for more information about filters to use such as the `GreaterThanFilter`.
```

Comment:
Reduced to one notice, and one remark that isn't in a callout block. The "see this for more information" doesn't need a callout.
Before this change it violated [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.